examples/interactivity/feature-enter-leave.html

```diff
@@ -5,11 +5,11 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta charset="UTF-8">
   <!-- Include CARTO GL JS -->
-  <script src="../../../dist/carto-gl.js"></script>
+  <script src="../../dist/carto-gl.js"></script>
```

Comment:
Please, no more directory changes, please

Reply:
This is development, bro